countdown/countdown.js

```diff
@@ -0,0 +1,50 @@
+var paused = false; 
+
+function getSeconds() {
+    hours = Number(document.getElementById("hours").value);
+    minutes = Number(document.getElementById("minutes").value);
+    seconds = Number(document.getElementById("seconds").value);
+    return ((hours * 60 * 60) + (minutes * 60) + seconds); 
```

Comment:
This was originally a one liner, but it looked really long and confusing, so i split it up